### PR TITLE
Labels annotations lettering

### PR DESCRIPTION
**Pull request by lynnerang**

### CODE REVIEW

File: src/Painter.ts [hunk 1]
```diff
@@ -501,24 +573,32 @@ const buildAnnotation = (options: {
...
     | 'keystop'
+    | 'label'
     | 'spacing'
     | 'style',
 }): {
   diamond: PolygonNode,
   rectangle: FrameNode,
   text: TextNode,
-  icon: FrameNode,
+  icon: FrameNode
```
Comment (gblakeman) [inline src/Painter.ts:583]: Not sure why this isn’t triggered in the linter, but any list like this should still have a comma at the end. It doesn’t hurt anything and saves time in the future if the list is modified.

File: src/Painter.ts [hunk 2]
```diff
@@ -2386,9 +2468,9 @@ export default class Painter {
     }
 
     // set up some information
-    const { annotationText } = nodeData;
-    const annotationType: 'keystop' = 'keystop';
-    const annotationName = `Keystop for ${this.node.name}`;
+    const annotationText = numberToLetters(+nodeData.annotationText);
```
Comment (gblakeman) [inline src/Painter.ts:2471]: Ignorant question: is the `+` just a shorthand to force a string into an `int` instead of using something like `.parseInt`?

**Review by gblakeman — APPROVED** [review]
🌈🦄

Great work! A couple questions for my clarification, but I don’t think they’ll lead to any changes.

File: src/Painter.ts (hunk 2 above)
Comment (lynnerang) [reply src/Painter.ts:2472]: yep!  I'd be happy to swap it if you think the parse method is more clear/readable though

**Pull request merged**